src/Tags/Vite.php

```diff
@@ -17,6 +17,13 @@ public function index()
             throw new \Exception('Please provide a source file.');
         }
 
-        return app(LaravelVite::class)($src);
+        $directory = $this->params->get('directory');
+        $hot = $this->params->get('hot');
+
+        return app(LaravelVite::class)
+            ->withEntryPoints($src)
+            ->useBuildDirectory($directory)
+            ->useHotFile($hot)
```

Comment:
`useHotFile` expects a full path. So you'd have to do `hot="/users/joshua/sites/yoursite/storage/vite.hot"` which is a bit awkward. Let's make it so that it'll prepend `base_path()` to whatever you pass, so you can do `hot="storage/vite.hot"`.

Maybe `->useHotFile($hot ? base_path($hot) : null)` will do it.